tests/test_post_load.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+
+import marshmallow
+
+import marshmallow_dataclass
+
+
+class TestFieldForSchema(unittest.TestCase):
+    def test_postload_a(self):
+        @marshmallow_dataclass.dataclass
+        class Person:
+            name: str
+
+            @marshmallow.post_load
+            def a(self, data, **_kwargs):
+                data["name"] = data["name"].capitalize()
+                return data
+
+        actual = Person.Schema().load({"name": "matt"})
+        expected = Person(name="Matt")
+        self.assertEqual(actual, expected)
+
+    def test_postload_z(self):
```

Comment:
My eyes might be failing me, but is there a difference between this test and `test_postload_a`?

Reply:
The only difference is the name of the post_load function. See https://github.com/lovasoa/marshmallow_dataclass/issues/75

Reply:
Oh, interesting--didn't think the post load functions would run in deterministic order. OK, it might make it clearer to have a single test with both schemas and sets of assertions and rename it to `test_post_load_method_naming_does_not_affect_data` or something like that. Could even add a code comment above it with

```python
# Regression test for https://github.com/lovasoa/marshmallow_dataclass/issues/75
```

so our future selves won't be confused by this.

No biggie, though--your call

Reply:
done